source/pele/cell_lists.h

```diff
@@ -592,7 +592,7 @@ void CellLists<distance_policy>::setup(Array<double> coords)
 
     // print messages if any of the parameters seem bad
```

Comment:
p.s. I was never happy with these efficiency warnings.  They didn't seem to work very well.  If you have a better idea of how to implement them feel free to shout out